go/kbnm/host_json.template

```diff
@@ -4,6 +4,7 @@
   "path": "@@HOST_PATH@@",
   "type": "stdio",
   "allowed_origins": [
+    "chrome-extension://ognfafcpbkogffpmmdglhbjboeojlefj/",
     "chrome-extension://kockbbfoibcdfibclaojljblnhpnjndg/",
     "chrome-extension://gnjkbjlgkpiaehpibpdefaieklbfljjm/"
```

Comment:
Not as JSON. :/

Could include our own comment syntax and strip them out during the preprocessing of the template, but adds more bug surface area on 3 separate/different scripts (linux, mac, windows).